src/button-dropdown/index.tsx

```diff
@@ -41,6 +41,8 @@ const ButtonDropdown = React.forwardRef(
       metadata: {
         mainAction: !!mainAction,
         checkboxItems: hasCheckboxItems(items),
+        hasDisabledReason: Boolean(disabledReason),
+        hasDisabledReasons: hasDisabledReasonItems(items),
```

Comment:
Just to avoid confusion when reading the analytics data, can we use a name that differentiates them better? 
Something like `hasDisabledReaons` and `haveItemsWithDisabledReasons`?

**Update**
Looking at the rest of the PR it seems that we use this consistently across the board. In this case I think we can ignore this comment.